service/access.go

```diff
@@ -347,6 +371,18 @@ func (a *accessService) updateAccessToken(ctx context.Context, domain, role, pro
 	return at.(*AccessTokenResponse), err
 }
 
+func accessCacheMemoryUsage(acd *accessCacheData) int64 {
+	tokenSize := int64(unsafe.Sizeof(acd.token)) + int64(len(acd.token))
+	domainSize := int64(unsafe.Sizeof(acd.domain)) + int64(len(acd.domain))
+	roleSize := int64(unsafe.Sizeof(acd.role)) + int64(len(acd.role))
+	proxySize := int64(unsafe.Sizeof(acd.proxyForPrincipal)) + int64(len(acd.proxyForPrincipal))
+	expiresInSize := int64(unsafe.Sizeof(acd.expiresIn))
+	expirySize := int64(unsafe.Sizeof(acd.expiry))
+	scopeSize := int64(unsafe.Sizeof(acd.scope)) + int64(len(acd.scope))
+
+	return tokenSize + domainSize + roleSize + proxySize + expiresInSize + expirySize + scopeSize
```

Comment:
identical implementation, can you check that they have the same output?
```suggestion
	structSize := int64(unsafe.Sizeof(*accessCacheData))
	stringSize := int64(len(acd.token) + len(acd.domain) + ...)
	
	return structSize + stringSize
```

Reply:
@WindzCUHK 
fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/8363817895e6a2447ba029bf48de8dcf8e404d52